### PR TITLE
update generated comp. files, 1.16

### PR DESCRIPTION
**Pull request by kbhawkey**
- removed two generated federation files

### CODE REVIEW

**Comment by kbhawkey** [comment]
@simplytunde , looking at the generated files, the kubelet page is not updated. There is a note
in reference.yml about manually updating this page.
@tengqm , if the `kubelet` page needs content changes, is it a case of manually running the cmd and creating the content from the output? I can modify the page to have the same front matter content; I have not tried to diff the output.

**Comment by tengqm** [comment]
@kbhawkey Sadly yes. We have to manually update the content for kubelet.

**Comment by kbhawkey** [comment]
The kubelet page is updated now, though surprised by the table layout. Let me know what you think.

**Comment by simplytunde** [comment]
kubelet page: https://5d7fc61831569b0008d65d3c--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/kubelet/


@kbhawkey It looks very different . @tengqm  any idea why it looks so different?

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 1]
```diff
@@ -800,7 +800,7 @@ kubelet [flags]
       <td></td><td style="line-height: 130%; word-wrap: break-word;">Comma-separated list of HTTP headers to use when accessing the url provided to --manifest-url. Multiple headers with the same name will be added in the same order provided. This flag can be repeatedly invoked. For example: --manifest-url-header 'a:hello,b:again,c:world' --manifest-url-header 'b:beautiful' (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)</td>
     </tr>
     <tr>
-      <td colspan="2">--master-service-namespace string<td>
+      <td colspan="2">--master-service-namespace string</td>
```
Comment (kbhawkey) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md]: @tengqm , Thanks for your help! :tada:

Comment (kbhawkey) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: Good, I spotted and fixed another typo and missed table markup.

**Comment by simplytunde** [comment]
Kubelet page is  looking good. 
/lgtm

**Pull request merged**